src/components/Cart-Page/Cart-Products.jsx

```diff
@@ -1,20 +1,23 @@
 import { Link } from "react-router-dom";
 import { useCartContext } from "../../hooks/cart/cart-context";
+import { useWishContext } from "../../hooks/wishList/wish-context";
 
 export const CartProduct = () => {
-    const { state,dispatch } = useCartContext();
-
+    const { state, dispatch } = useCartContext();
+    const { wishState, wishDispatch } = useWishContext();
+    let {addToishListMessage}=useWishContext();
     return <div>
-
-        {state.cartItems.length === 0 ? <div>No products availavle in cart.</div> :
+        <div className="h3">total items in cart: {state.cartCount}</div>
+        {state.cartItems.length === 0 ? <div className="empty-message"> No products availavle in cart.</div> :
 
             <div className="product-summary">
+
                 <div className="product-content">
                     {state.cartItems.map((items) => {
-                  
+
                         return <div className="card text-overlay-card no-footer" key={items.id}>
                             <div className="img-div">
-                                <img src="https://engin-ui.netlify.app/documentation/landing-components/card/image.jpg" alt="product-img" />
+                                <img src={items.img} alt="product-img" />
```

Comment:
You can get the alt text from api for different items, rather than using static text for all images.